Editor/EditorWindow/OnionDataEditor.cs

```diff
@@ -143,7 +144,14 @@ public static void OpenWithOnionDataEditor()
             }
         }
 
-
+public static void OpenWithOnionDataEditor(Object selectObj)
```

Comment:
Looks pretty good, but it need Indent.

Reply:
Ah, yes, that is my bad, I unindented so that I could copy/paste it easier into my github post without the extra spacing and forgot to put it back.